### PR TITLE
Funmooc fix contributors filter and rename new courses filter

### PR DESCRIPTION
**Pull request by sampaccoud**
### Fixed

- Fix contributors filter by overriding only its human name
- Rename "New course" option to "New courses" for coherence with filter name

### CODE REVIEW

File: sites/funmooc/src/backend/funmooc/settings.py [hunk 1]
```diff
@@ -480,9 +480,9 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
                 "is_autocompletable": True,
                 "is_searchable": True,
                 "min_doc_count": 0,
-                "name": "contributors",
+                "name": "persons",
                 "position": 6,
-                "reverse_id": "contributors",
+                "reverse_id": "persons",
```
Comment (mbenadda) [inline sites/funmooc/src/backend/funmooc/settings.py:485]: If you change the `reverse_id` here you also need to change the reverse ID on the live site, right?

Comment (sampaccoud) [reply sites/funmooc/src/backend/funmooc/settings.py:485]: Yes ! That's the plan if I don't forget :sweat_smile:

**Review by mbenadda — APPROVED** [review]
👍

File: sites/funmooc/src/backend/funmooc/settings.py (hunk 1 above)
Comment (mbenadda) [reply sites/funmooc/src/backend/funmooc/settings.py:485]: Just making sure it was intentional :)

You'll also need to re-generate the ES indices.

**Pull request merged**